modules/tin_score/tin_score.py

```diff
@@ -0,0 +1,123 @@
+###############################################################################
+#
+#   Main script which does the job of parsing values and plotting graphs
+#   for tin scores
+#
+#   AUTHOR: Krish Agarwal
+#   AFFILIATION: University_of_Basel
+#   CONTACT: akrish136@gmail.com
+#   CREATED: 1-11-2020
+#   LICENSE: Apache_2.0
+#
+###############################################################################
+
+from __future__ import print_function
+import logging
+import os
+
+from multiqc.plots import linegraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    This class is instantiated in setup.py file and contains
+    all the functions required by the tin-score plugin.
+    """
+
+    def __init__(self):
+        """
+        Initializes the parent function of MultiQC and calls
+        relevant functions for the searching of files and the
+        plotting of graphs.
+        """
+        super(MultiqcModule, self).__init__(
+            name="tin-score",
+            anchor="tin-score",
+            href="",  # to add
+            info="",  # to add
```

Comment:
?

Reply:
Could you point me to a description, opening  https://git.scicore.unibas.ch/zavolan_group/tools/tin_score_calculation asks me to sign up and take permission.

Reply:
Hmm, that is strange, I thougth it is open...
*Given a set of BAM files and a gene annotation BED file, calculates the Transcript Integrity Number (TIN) for each transcript.*

Reply:
Once this is in please check the code once again and if everything is fine I will merge.

Reply:
Yes, done!